releng/OWNERS

```diff
@@ -3,6 +3,7 @@
 approvers:
 - calebamiles
 - justaugustus
+- Katharine
```

Comment:
/assign @Katharine

Reply:
(already assigned :) )

Reply:
stephen you're ruining my gubernator 0 attempt :upside_down_face:

Reply:
Oh yeah? :stuck_out_tongue_winking_eye: